Source/ReferenceConnectorWorkerService/ReferenceConnectorWorkerService.cs

```diff
@@ -54,8 +55,14 @@ private void CreateSubmitPipeline()
 
             _aggregationSubmitPipeline = aggregationFilterPipelineElement;
 
-            var httpSubmitBinaryElement = new HttpSubmitBinaryPipelineElement(null);
-            httpSubmitBinaryElement.ApiClientFactory = apiClientFactory;
+            var circuitBreakerOpts = new CircuitBreakerOptions();
+            var httpSubmitBinaryElement = new DirectSubmitBinaryPipelineElement(null)
+            {
+                ApiClientFactory = apiClientFactory,
+                CircuitProvider = new AzureBlobRetryProviderWithCircuitBreaker(circuitBreakerOpts, true),
+                RetryProvider = new AzureBlobRetryProviderWithCircuitBreaker(circuitBreakerOpts, true),
+                // Log = optional logger

```

Comment:
question (non-blocking)
Are we leaving this line as a reminder to add logging in a future iteration? If we still need to leave a comment, might be also good to add a reason why or if there is already a task assigned for this then the ID would be handy, just some options here.

Reply:
This is a hint to the connector author that a logger may be passed to the pipeline.  I agree it would be great for this demo code to actually implement it.  I'll ask around to see if that should go on the backlog.